source/opt/fold.cpp

```diff
@@ -289,5 +290,54 @@ bool IsFoldableConstant(const analysis::Constant* cst) {
     return cst->AsNullConstant() != nullptr;
 }
 
+ir::Instruction* FoldInstruction(
+    ir::Instruction* inst,
+    std::function<uint32_t(uint32_t)> id_map ) {
+  if (!IsFoldableOpcode(inst->opcode())) {
```

Comment:
Why is this not `if (!inst->IsFoldable()) { return false; }`

Suggest a comment here.

Reply:
done.